### PR TITLE
update doc link

### PR DESCRIPTION
**Pull request by jerrywgz**
Link all previous detection models to PaddleDetection.

### CODE REVIEW

File: fluid/PaddleCV/object_detection/README_quant.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ Hi!
 
 This directory has been deprecated.
 
-Please visit the project at [PaddleCV/object_detection](../../../PaddleCV/object_detection).
+Please visit the project at [PaddleCV/PaddleDetection](../../../PaddleCV/PaddleDetection).
```
Comment (qingqing01) [inline fluid/PaddleCV/object_detection/README_quant.md]: 这个是在 ../../../PaddleCV/ssd 中

Comment (jerrywgz) [reply fluid/PaddleCV/object_detection/README_quant.md]: rcnn yolo和object detection都放到link到各自PaddleCV的目录里吗

Comment (qingqing01) [reply fluid/PaddleCV/object_detection/README_quant.md]: 只有 README_quant.md

Comment (jerrywgz) [reply fluid/PaddleCV/object_detection/README_quant.md]: Done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**